### ISSUE TITLE
Cisco Support DEAD  (MERAKI = Linux) ?  

### ISSUE THREAD

**Issue by cbrede** [Azure/Azure-Sentinel#10371]
I downloaded the integration script an Workbook from

https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/CiscoMeraki/Parsers/CiscoMeraki.txt

also

https://github.com/Azure/Azure-Sentinel/issues/9471

and

https://documentation.meraki.com/General_Administration/Monitoring_and_Reporting/Syslog_Server_Overview_and_Configuration 
https://documentation.meraki.com/General_Administration/Monitoring_and_Reporting/Syslog_Event_Types_and_Log_Samples

is totally broken

1.  base on old linux proxy (deprecated)
2. no IPv6 support 
3. changes in firewall extenstion  (rules# or policy) 1,2,3 .... 
4. rules are not accept by "cellular_firewall,vpn_firewall"
5. no ......

is someone responsible for this ?????????????????

**Comment by v-sudkharat** [comment]
Hey @cbrede, Thank you for highlighting this issue with us. This is an enhancement to existing parser, and our team already noted down this issue. Currently the updated parser is in testing face, once it clears, we will merge it into Master. 
Meanwhile, sharing the parser file by reference(https://github.com/Azure/Azure-Sentinel/issues/9471) with you to get unblocked you from this issue: - 
[CiscoParser.txt](https://github.com/Azure/Azure-Sentinel/files/15091832/CiscoParser.txt)

Thanks!

**Comment by v-sudkharat** [comment]
@cbrede. Once the parser merged into the master, the changes will reflect into the upcoming solution version. So, closing this issue from GitHub. If you still need support for this issue, feel free to re-open it any time. Thank you for your co-operation.
